crates/red_knot_python_semantic/src/types.rs

```diff
@@ -3104,6 +3104,12 @@ impl<'db> Type<'db> {
                 Some(KnownClass::ParamSpec) => Ok(todo_type!(
                     "Support for `typing.ParamSpec` instances in type expressions"
                 )),
+                Some(KnownClass::ParamSpecArgs) => Ok(todo_type!(
+                    "Support for `typing.ParamSpecArgs` instances in type expressions"
+                )),
+                Some(KnownClass::ParamSpecKwargs) => Ok(todo_type!(
+                    "Support for `typing.ParamSpecKwargs` instances in type expressions"
+                )),
```

Comment:
I think it's somewhat likely that all of these TODOs would be resolved at once -- they are really a single feature. So I'd be inclined to simplify this to:

```suggestion
                Some(KnownClass::ParamSpec | KnownClass::ParamSpecArgs | KnownClass::ParamSpecKwargs) => Ok(todo_type!(
                    "Support for `typing.ParamSpec`"
                )),
```